doc/modules/ensemble.rst

```diff
@@ -150,6 +150,16 @@ In contrast to the original publication [B2001]_, the scikit-learn
 implementation combines classifiers by averaging their probabilistic
 prediction, instead of letting each classifier vote for a single class.
 
+Comparison of Histogram-based Gradient Boosting Trees over Random Forests in terms of computational cost:
+
+   - Building shallow trees: HGBT can be computationally more efficient than RF when building shallow trees because it only needs to consider a limited number of bins to construct the splits, whereas RF builds deep trees that require more iterations to reach optimal splits.
+
+   - Sequential boosting: In HGBT, the trees are built sequentially, with each tree correcting the mistakes of the previous trees. This can be more computationally efficient than RF, where all the trees are built independently and in parallel.
```

Comment:
> The phrase "This can be more computationally efficient than RF" is a bit vague, I suggest giving a deeper explanation on this regard:

Made the changes as requested